### PR TITLE
[office-js][office-js-preview] (Excel) Pulling latest Excel type definitions from CDN

### PR DESCRIPTION
**Pull request by alison-mk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
Release URL: https://appsforoffice.microsoft.com/lib/1.1/hosted/office.debug.js
Beta URL: https://appsforoffice.microsoft.com/lib/beta/hosted/office.debug.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

File: types/office-js-preview/index.d.ts
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:28376]: Note to reviewers, I checked with eng to confirm this change.

File: types/office-js/index.d.ts
Comment (alison-mk) [inline types/office-js/index.d.ts:27317]: Note to reviewers: I checked with eng to confirm this change.

File: types/office-js/index.d.ts
Comment (AlexJerabek) [inline types/office-js/index.d.ts:28520]: Looks like these changes reverted back. I don't think it's a big deal to leave it until the changes I made on 10/14 make it in.

File: types/office-js-preview/index.d.ts
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts:71142]: This line and its duplications have been reverted. The source change was made on 10/14, so it's possible this is overwriting a local change and the fix is coming soon.

**Review by AlexJerabek — APPROVED** [review]
Approving with a couple reversions to investigate.

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:50214]:
```suggestion
         * The Comments field in the metadata of the workbook. These have no connection to comments by users made in the workbook.
```

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:54562]:
```suggestion
         * @param base64ImageString A Base64-encoded string representing an image in either JPEG or PNG format.
```

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:54905]:
```suggestion
         * Converts the shape to an image and returns the image as a Base64-encoded string. The DPI is 96. The only supported formats are `Excel.PictureFormat.BMP`, `Excel.PictureFormat.PNG`, `Excel.PictureFormat.JPEG`, and `Excel.PictureFormat.GIF`.
```

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:54914]:
```suggestion
         * Converts the shape to an image and returns the image as a Base64-encoded string. The DPI is 96. The only supported formats are `Excel.PictureFormat.BMP`, `Excel.PictureFormat.PNG`, `Excel.PictureFormat.JPEG`, and `Excel.PictureFormat.GIF`.
```

File: types/office-js/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:28520]:
```suggestion
     * @param base64File Optional. The Base64-encoded .xlsx file. The default value is null.
```

File: types/office-js/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:34002]:
```suggestion
         * Renders the range as a Base64-encoded PNG image.
```

File: types/office-js/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:50158]:
```suggestion
         * Creates an image from a Base64-encoded string and adds it to the worksheet. Returns the `Shape` object that represents the new image.
```

File: types/office-js/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:50163]:
```suggestion
         * @param base64ImageString A Base64-encoded string representing an image in either JPEG or PNG format.
```

File: types/office-js/index.d.ts
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:50485]:
```suggestion
         * Converts the shape to an image and returns the image as a Base64-encoded string. The DPI is 96. The only supported formats are `Excel.PictureFormat.BMP`, `Excel.PictureFormat.PNG`, `Excel.PictureFormat.JPEG`, and `Excel.PictureFormat.GIF`.
```

File: types/office-js/index.d.ts
Comment (alison-mk) [reply types/office-js/index.d.ts:28520]: Ok, thanks! No, the 10/14 changes are not in the source yet.

File: types/office-js-preview/index.d.ts
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:71142]: Ok, thanks! No, the 10/14 changes are not in the source yet.

File: types/office-js-preview/index.d.ts
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:29312]: How can we check the downstream rendering? Wait until this is merged into reference and then see what it looks like?

File: types/office-js-preview/index.d.ts
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:22145]: Good catch! This looks like an eng error that I didn't catch in the design review. Do you think it will cause any issues downstream?

File: types/office-js-preview/index.d.ts
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:22414]: Thank you for going through these new type definitions so thoroughly! Really appreciate your effort. 

I'm not going to incorporate the contraction suggestions because, even though they're more aligned with our style guide, they're not grammatically incorrect and the overhead for making all the changes is pretty high (the process that generates these definitions in particular is challenging and outside the regular process). I did incorporate all your other suggestions!

File: types/office-js-preview/index.d.ts
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:29878]: Good catch. I will check with the engineer about this.

File: types/office-js-preview/index.d.ts
Comment (ElizabethSamuel-MSFT) [reply types/office-js-preview/index.d.ts:29312]: Yes, that's certainly an option

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Review by andrewbranch — APPROVED** [review]

**Review by andrewbranch — APPROVED** [review]

File: types/office-js-preview/index.d.ts
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:22414]: Heads up, I was able to include your contraction suggestions when the engineer made other changes in the source code! They'll make it to DefinitelyTyped eventually.

**Pull request merged**
